CMake/Dependencies.cmake

```diff
@@ -118,9 +118,19 @@ else()
 endif()
 
 if(NOT NOSOUND)
-  dependency_options("SDL_audiolib" DEVILUTIONX_SYSTEM_SDL_AUDIOLIB OFF DEVILUTIONX_STATIC_SDL_AUDIOLIB)
+  if(NOT DEFINED DEVILUTIONX_SYSTEM_SDL_AUDIOLIB)
+	find_package(SDL_audiolib QUIET)
+    if(SDL_audiolib_FOUND)
+      message("-- Found SDL_audiolib")
+    else()
+      message("-- Suitable system SDL_audiolib package not found, will use SDL_audiolib from source")
+      set(DEVILUTIONX_SYSTEM_SDL_AUDIOLIB OFF)
+    endif()
+  endif()
+  dependency_options("SDL_audiolib" DEVILUTIONX_SYSTEM_SDL_AUDIOLIB ON DEVILUTIONX_STATIC_SDL_AUDIOLIB)
   if(DEVILUTIONX_SYSTEM_SDL_AUDIOLIB)
     find_package(SDL_audiolib REQUIRED)
+    include_directories(${SDL_audiolib_INCLUDE_DIR})

```

Comment:
The imported package already defines `INTERFACE_INCLUDE_DIRECTORIES`, this is not necessary.